app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,56 @@
+require 'report_email_sender'
```

Comment:
(The problems with using require directly in rails are detailed here: http://guides.rubyonrails.org/autoloading_and_reloading_constants.html#autoloading-and-require)